reference/7.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -319,21 +319,29 @@ A cast can also be performed when a variable is assigned to using [cast notation
 
 #### Comma operator `,`
 
-As a binary operator, the comma creates an array. As a unary operator, the
-comma creates an array with one member. Place the comma before the member.
+As a binary operator, the comma creates an array or appends to the array being
+created. In expression mode, as a unary operator, the comma creates an array
```

Comment:
```suggestion
created. In expression mode, or as a unary operator, the comma creates an array
```

Reply:
The point is, the unary comma is a parse error in argument mode.

Reply:
I do see that and I was trying (poorly) to ensure that the unary operator portion still existed outside of  the expression mode as both `(,1)` in expression mode and `,1` outside expression mode use it as a unary operator. The way your change reads only references it inside expression mode siloing it.